SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialGameInstance.h

```diff
@@ -66,4 +72,7 @@ class SPATIALGDK_API USpatialGameInstance : public UGameInstance
 	// If this flag is set to true standalone clients will not attempt to connect to a deployment automatically if a 'loginToken' exists in arguments.
 	UPROPERTY(Config)
 	bool bPreventAutoConnectWithLocator;
+
+	UPROPERTY()
+	USpatialLatencyTracer* SpatialLatencyTracer = nullptr;
```

Comment:
Should `SpatialLatencyTracer` be recreated when server travelling or destroying `Spatialconnection`?

Reply:
I don't think we need to. We might want to measure map travel events at some point, so it would be useful to have them persist beyond the net driver.